### ISSUE TITLE
Boundaries should be easily un-anchored from physical objects 

### ISSUE THREAD

**Issue by jjiglesiasg** [openstreetmap/iD#7619]
Boundaries are polytical NON existing objects that should be easily detached from actual physical objects like roads, rivers, etc.

The disconnection feature of the right click should be applicable for large sections of these "forced" juntions instead of node by node. Indeed roads/waterways and boundaries should not be possible to be connected at all. Due to different nature, and boundaries not being a real object.

Thanks

JJ

**Comment by quincylvania** [comment]
@jjiglesiasg You can click-and-drag on the map while holding shift to lasso all the connection nodes and then use the Disconnect command to detach them all at once. Perhaps this is sufficient?

**Comment by maro-21** [comment]
> roads and boundaries should not be possible to be connected at all

I don't agree. I had a situation when I had to connect a road to a boundary because of the names of the streets in two cities. If I didn't correct, overpass would find the name of a street which belongs to the other city.

**Comment by jjiglesiasg** [comment]
Hi quincylvania. there is no possibility to disconnect multiple nodes because boundaries are part of relations and ID do not allow to disconnect following your example.

maro-21 I dont know under what exact circunstances your example falls, but again Boundaries are NOT existing objects and roads, waterways, etc, are REAL objects that needs to be adjusted, corrected, and redrawn, fighting with hundred of nodes connected with boundaries do Not make sense at all and for worst if they could not be easily detached.

Likewise power features, should not be possible to connect a road/waterway node to a power line node, they are objects of different nature...

BsRgds JJ

**Comment by DujaOSM** [comment]
I propose that selecting a way (open or closed) and issuing "disconnect" (via context menu or D key) disconnects all its nodes from all adjacent ways, except for those that it shares a relation with. That would be a huge time saver.

 On many occasions, I've encountered good-faith but clueless mapping where everything is glued together and it's almost impossible to make any changes in geometry of objects without breaking something. Think a single node adjoining three administrative boundaries, four highways and two farmland polygons: try untangling that. In fact, now that iD has a powerful validation engine, I would like having a warning against gluing objects from different "layers" (e.g. boundaries, waterways, landuse and highways).

**Comment by jjiglesiasg** [comment]
That's my bottom point, a mess all these connected areas or Political (Administrative) Boundaries to a way or river do not make sense at all for me at least...

**Comment by kymckay** [comment]
Validator discussion here has some relevance to #6631

Original issue might be addressed by now implemented #7652

**Comment by quincylvania** [comment]
@SilentSpike #7652 should help in some circumstances, but I suspect there might be something else at play here.

@jjiglesiasg Could you link to a map location where you're seeing the issue?

**Comment by DujaOSM** [comment]
@quincylvania I tried #7652 and, while I like the principle very much, it has a severe practical limitation: "This can't be disconnected because not enough of it is currently visible". In the case of administrative and other boundaries -- which are inevitably several screenfuls large -- this limitation makes it practically unusable.

As a test, try untangling this National Park boundary from the forest:
https://www.openstreetmap.org/edit?way=790335951#map=16/45.1471/19.4575

The best workaround I found is to select a node and do PgUp+D many times, but it has disadvantages that it unglues all adjacent ways, and it tends to wander off the desired way onto its neighbor.

**Comment by 1ec5** [comment]
> maro-21 I dont know under what exact circunstances your example falls, but again Boundaries are NOT existing objects and roads, waterways, etc, are REAL objects that needs to be adjusted, corrected, and redrawn, fighting with hundred of nodes connected with boundaries do Not make sense at all and for worst if they could not be easily detached.

Sure, a `boundary` relation is an abstraction of the various boundary markers, welcome signs, and changes in pavement that occur along the boundary. So too is a `highway` an abstraction of the road’s centerline, more or less. In one of the regions where I map, it’s very common for a road to be intentionally built along a boundary or a boundary to be defined in terms of the road’s centerline. For example:

* The border between Cincinnati and its suburb Fairfax is legally defined to run along the centerline of [this road](https://www.openstreetmap.org/way/287786500). I can’t say for certain that the `highway` way precisely corresponds to the legal centerline, because that would depend on which imagery layer you trust more, but it’s easy to verify that the road has a different name on either side of the road, because each city gives the road a different name.
* [This road](https://www.openstreetmap.org/way/17534407) runs along the border between two counties. Either county assigns a completely different name to the road. Sometimes roads like this include the county name. It would be weird for “[Defiance County Road A](https://www.openstreetmap.org/way/8769204)” to lie in Paulding County according to Overpass or Nominatim.
* [This road](https://www.openstreetmap.org/way/229119243) runs along the border between two townships. In this county, the townships coordinate their road numbers to match along township lines. However, each township has a different design for its highway shields (green on one side, white on the other). A [shield renderer](https://github.com/kennykb/osm-shields/) may need to display both for recognizability.

I’m not suggesting that a boundary should always be attached to nearby features. (Indeed, I’m always on the lookout for mappers incorrectly joining boundaries to the [Ohio River](https://wiki.openstreetmap.org/wiki/Ohio_River).) But there are enough legitimate cases for joining boundaries to roads. The alternatives would be less palatable:

* Make the road a member of the boundary relations. This would address your concern about fighting with nodes, but we stopped doing that in my area a long time ago, because a way simultaneously tagged as two very different kinds of things doesn’t work well with editor presets, and it can be a headache for some renderers too.
* Draw the road as a dual carriageway, even though there’s only a single carriageway.
